h/static/scripts/widget-controller.js

```diff
@@ -312,9 +347,11 @@ module.exports = function WidgetController(
     streamer.reconnect();
   });
 
-  // When a direct-linked annotation is successfully anchored in the page,
-  // focus and scroll to it
   $rootScope.$on(events.ANNOTATIONS_SYNCED, function (event, tags) {
```

Comment:
To provide a bit more context here, going forwards, I'd like to get to a place where these kinds of rules can be implemented in a more consistent and straightforward way that follows the [basic principles](http://redux.js.org/docs/introduction/ThreePrinciples.html) for a Redux app. Why? Because it makes these kinds of business rules easy to see from the code and test.

This means that all the "when X happens" cases should become actions (calls to `store.dispatch()` in annotationUI. eg. The current group changed, annotations finished loading, annotation anchoring finished) and all the consequences (eg. "tab Y should be selected") are state updates in the reducer functions.

The simpler we can make the logic now, the easier it will be to refactor to this in future.